scripts/docker/cpp-build-env.dockerfile

```diff
@@ -0,0 +1,30 @@
+# build:          docker build -t test_build_20 .
+# run:            docker run -it --rm test_build_20:latest
+# versioning:     docker tag cpp-build-env ethereum/cpp-build-env:v0.8.29.2017
+# upload:         docker push tcsiwula/cpp-build-env
+
+# get latest stable debian release image -- see https://hub.docker.com/_/debian/
+from debian:latest
```

Comment:
1. It should be `FROM`.
2. Do not use rolling version. Try sid or stretch.

Reply:
Went for stretch :)